chapter16/template.yaml

```diff
@@ -1,12 +1,123 @@
 ---
 AWSTemplateFormatVersion: '2010-09-09'
 Description: 'AWS in Action: chapter 16'
-Parameters:
-  KeyName:
-    Description: 'Key Pair name'
-    Type: 'AWS::EC2::KeyPair::KeyName'
-    Default: mykey
+Mappings:
+  RegionMap:
+    'ap-northeast-1':
```

Comment:
the nodejs cod hardcodes the region to us-east-1. Either fix that or get rid of the region map.